### PR TITLE
[HttpFoundation] Add support for sending raw cookies in the response

### PR DESCRIPTION
**Pull request by jakzal**
| Q | A |
| --- | --- |
| Branch | master |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #13646 |
| License | MIT |
| Doc PR | https://github.com/symfony/symfony-docs/pull/6436 |

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/Tests/CookieTest.php [hunk 1]
```diff
@@ -151,4 +151,13 @@ public function testToString()
         $cookie = new Cookie('foo', 'bar', 0, '/', '');
         $this->assertEquals('foo=bar; path=/; httponly', $cookie->__toString());
     }
+
+    public function testRawCookie()
```
Comment (javiereguiluz) [inline src/Symfony/Component/HttpFoundation/Tests/CookieTest.php]: Should we also add a test which creates a raw cookie for a request and then check if its contents are really not encoded?

Comment (jakzal) [inline src/Symfony/Component/HttpFoundation/Tests/CookieTest.php]: I don't know the way to capture headers in a test. Do you have an idea how to do it?

Comment (javiereguiluz) [inline src/Symfony/Component/HttpFoundation/Tests/CookieTest.php]: I'd say you can't. We'd need a functional test.

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by javiereguiluz** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @jakzal.

**Comment by murat11** [comment]
Can this be merged to lower versions? 2.8

**Pull request merged**